src/main/java/by/exadel/internship/entity/Interview.java

```diff
@@ -27,17 +27,19 @@
     @Column(name = "i_id", updatable = false, nullable = false)
     private UUID id;
 
-    @Column(name = "i_admin_id")
-    private UUID admin;
+    @JoinColumn(name = "i_admin_id")
+    @ManyToOne(cascade = {CascadeType.PERSIST,CascadeType.REFRESH,CascadeType.MERGE})
```

Comment:
Why do you need refresh here and in the techSpecialist?

Reply:
Fixed. I don't know why i added this cascadeType

Reply:
Still see REFRESH for the admin property

Reply:
Fixed